portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -1293,14 +1293,40 @@ definition {
 	}
 
 	@description = "Verify that a Text field's Predefined Value can be translated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test TextFieldPredefinedValueCanBeTranslated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133494 TextFieldPredefinedValueCanBeTranslated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		FormFields.editPredefinedValueInput(fieldValue = "Predefined value");
+
+		Form.addTranslation(localizedLanguage = "Spanish (Spain)");
+
+		FormFields.editPredefinedValueInput(fieldValue = "Valor predefinido");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
```

Comment:
Hi @fmarcio , this step doesn't seem to be necessary since you're not validating the predefined value for english.

Reply:
> Hi @fmarcio , this step doesn't seem to be necessary since you're not validating the predefined value for english.

@mlan0 actually this step is necessary because it is needed to click on the en-US flag and then change the language :-)